packages/govuk-frontend/src/govuk/settings/_typography-font.scss

```
@@ -27,15 +25,13 @@ $govuk-font-family-print: sans-serif !default;
 
 /// Include the default @font-face declarations
 ///
-/// If you have set $govuk-font-family to something other than
-/// `$govuk-font-family-gds-transport` this option is disabled by default.
+/// Defaults to true if "GDS Transport" appears in the $govuk-font-family
+/// setting.
 ///
 /// @type Boolean
 /// @access public
 
-$govuk-include-default-font-face: (
-  $govuk-font-family == $govuk-font-family-gds-transport
-) !default;
+$govuk-include-default-font-face: if(index($govuk-font-family, "GDS Transport"), true, false) !default;
```

Can we check for the double-quoted `"GDS Transport"` rather than just `GDS Transport`?

You never know, don't want to match future fonts:

```
GDS Transport
GDS Transport New
```

```suggestion
$govuk-include-default-font-face: if(index($govuk-font-family, '"GDS Transport"'), true, false) !default;
```

---

This is using `list.index` rather than string matching ([`str-index`](https://sass-lang.com/documentation/modules/string/#index)), so should only match complete entries.

Double checked by changing to `if(index($govuk-font-family, '"GDS"')` which caused the tests to fail (always `false`).

---

Nice. Good to know 🚀